### ISSUE TITLE
Openmodelica does not compile on Gentoo - using system zlib

### ISSUE THREAD

**Issue by leo-recover** [OpenModelica/OpenModelica#9269]
### Description

When trying to upgrade from 1.18.0 to 1.19.0, OpenModelica does not compile anymore on my Gentoo installation. The error log is the same as the issue I described some time ago [here](https://trac.openmodelica.org/OpenModelica/ticket/6148): the compiler is using the system zlib, while the bundled one should be used. Back then the issue was fixed by pull request #6799.

I was trying to investigate what changed, and I noticed commit 3378793e98e0a754e92a1f0b7005f2b3312ba306 in 3rdParty
So I changed OMCompiler/Compiler/runtime/Makefile.in as below:

`CPPFLAGS = @CPPFLAGS@ -I$(OMC_CONFIG_INC) -I$(top_builddir)/SimulationRuntime/c -I$(top_builddir)/SimulationRuntime/c/simulation/results -I$(top_builddir)/SimulationRuntime/c/util -I$(top_builddir)/SimulationRuntime/c/meta -I$(top_builddir)/SimulationRuntime/c/meta/gc -I. $(CORBAINCL) $(GCINCLUDE) -I$(FMIINCLUDE) -I$(GRAPHINCLUDE) -I$(CJSONINCLUDE) -I$(ZMQINCLUDE)  -I"$(TOP_DIR)/3rdParty/zlib" -I"$(TOP_DIR)/3rdParty/libffi/install/include/"`

Basically referencing the zlib folder included in the 3rdParty directory, instead of the one in FMIL, and now omc is compiling again.

### Version and OS

 - OpenModelica Version: 1.19.0
 - OS: Gentoo

**Comment by casella** [comment]
@leo-recover mind making a PR with your patch?

Thanks!
